README.md

```diff
@@ -4,9 +4,9 @@ The platform assists member in managing expenses, tasks , voting, and more.
 
 ## ERD
 
-![ERD](https://user-images.githubusercontent.com/74178515/204143707-970216f5-e382-47d1-ab2e-209aa7aaa18b.png)
+![ERD](https://lucid.app/documents/embedded/af373954-3144-4325-9253-494a0af1512b?invitationId=inv_d92b11b0-d0dd-445e-98c2-8493166012ea#)
```

Comment:
About the Commune I would: 
- add a field of "description"
- remove the User_id, chore_id, vote_id from the commune.

what is the wallet? 

About the User I would:
- add username
- add (FK)commune_id

About the Chore I would:
- remove the vote_id 
- add "has_passed" 
- add (FK)commune_id
- maybe we need to add status - to know if the chore started/was completed.

About the Vote I would:
- remove the "had_passed"
- add "in_favor_of"

Reply:
The purpose of wallet in Commune is to know how much money the commune have (to know how much money they can waste on different chores etc).

I added password as well as username in User.